zulipterminal/model.py

```diff
@@ -930,7 +934,8 @@ def _register_desired_events(self, *, fetch_data: bool=False) -> str:
             'update_message_flags',
             'muted_topics',
             'realm_user',  # Enables cross_realm_bots
-            'realm_user_groups'
+            'realm_user_groups',
+            'zulip_version'
```

Comment:
Also worth adding a comment that this is present by default from one particular `feature_level`?